src/internal/connector/onedrive/item.go

```diff
@@ -51,9 +52,13 @@ func driveItemReader(
 
 	downloadURL := item.GetAdditionalData()[downloadURLKey].(*string)
 
-	// TODO: We should use the `msgraphgocore` http client which has the right
-	// middleware/options configured
-	resp, err := http.Get(*downloadURL)
+	clientOptions := msgraphsdk.GetDefaultClientOptions()
+	middlewares := msgraphgocore.GetDefaultMiddlewaresWithOptions(&clientOptions)
+
+	httpClient := msgraphgocore.GetDefaultClient(&clientOptions, middlewares...)
+	httpClient.Timeout = time.Second * 90
```

Comment:
Default is 30, but thought I would keep it same as https://github.com/alcionai/corso/blob/msgraphcore-http-client/src/internal/connector/graph/service_helper.go#L53